core/storage/user/gae_models.py

```diff
@@ -2014,8 +2014,8 @@ def increment_score_for_user(cls, user_id, score_category, increment_by):
             model.put()
 
 
-class UserCommunityRightsModel(base_models.BaseModel):
-    """Model for storing user's rights on community dashboard.
+class UserContributionRightsModel(base_models.BaseModel):
```

Comment:
Do we need to delete UserCommunityRightsModel models that exist in test/prod server?